tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -52,3 +52,15 @@ func ReplaceVmFromVmgroup(ip, name, vmName string) ([]byte, error) {
 	log.Printf("Replacing VM [%s] from a vmgroup [%s] on esx [%s] \n", vmName, name, ip)
 	return ssh.InvokeCommand(ip, admincli.ReplaceVmFrmVmgroup+"--name="+name+" --vm-list="+vmName)
 }
+
+// Initialize the (local) Single Node Config DB
+func ConfigInit(ip string) ([]byte, error) {
```

Comment:
Yes, I thought about this while writing SingleNode util but since we do not need it for this test I didn't add it. I can address this later on.